playbook.yml

```diff
@@ -2,9 +2,23 @@
   gather_facts: no
   vars:
     project_name: "{{ meta.namespace }}"
-    database_connection:
+    database_settings:
       username: pulp
       password: pulp
+    pulp_default_settings:
+      databases:
+        default:
+          HOST: postgres
+          ENGINE: django.db.backends.postgresql_psycopg2
+          NAME: pulp
+          USER: pulp
```

Comment:
I still do not think these should be explicitly defaulted to. I think we need something like:
`USER: "{{ database_settings.username }}"`

(We do not need the "default" filter because database_settings is always defined.)

And if the user really wants to override it in the pulp_settings data structure, they can.